frontend-client/app/_components/NavLink.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+interface NavLinkProps {
+  href: string
+  text: string
+}
+
+export default function NavLink({ href, text }: NavLinkProps) {
+  const pathname = usePathname()
+
+  return (
+    <Link
+      href={href}
+      className={`text-lg hover:opacity-60 ${
+        pathname === href ? 'text-primary' : ''
+      }`}
```

Comment:
사소하긴 한데 `cn` 함수로 적는 게 더 깔끔한 것 같습니다!

```suggestion
      className={cn(
        'text-lg hover:opacity-60',
        pathname === href && 'text-primary'
      )}
```

Reply:
확인하고 적용했습니다!